frontend/src/components/common/ProductDisplay/ProductList/Product/Product.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './Product.scss';
+
+interface IProp {
+  isLoading: boolean;
+  id: number;
+  name: string;
+  price: number;
+  imageUrl: string;
+}
+
+function Product(props: IProp) {
+  const { isLoading, name, price, imageUrl } = props;
+  console.log("loading: ", isLoading);
+  return (
+    <>
+      <section>
+        <div className="product">
+          <div className="product__container ">
+            <img className={`product__image ${isLoading ? 'loading' : ''}`} src={imageUrl} alt="product" />
+            <div className="product__info">
+              <div className="product__info-name">{name}</div>
+              <p className="product__info-price">{price} $</p>
```

Comment:
I suggest you should write this as a helper function. because this might be reused in many other places. if we suddenly want to the value of product price to show € instead of $. you just need to change the helper function. also it is easier to write UT for that.
Ex: **renderProductPrice** in **helper/product.ts**